Header header
# subject vehicle has position (0,0)

# movement (position, heading, speed, acceleration) of the subject vehicle
ObjectMovement vehicle_movement